Dockerfile.template

```diff
@@ -1,6 +1,11 @@
 # vim:set ft=dockerfile:
 FROM ubuntu:%%SUITE%%
 
+ARG SETUP_SCRIPT=https://downloads.mariadb.com/MariaDB/mariadb_repo_setup
```

Comment:
Can we simplify the `ARG`s here down to a single one, probably `SETUP_SCRIPT`. The default build of the community image without any `ARG`s specified is needed for it to flow through the CI correctly.

There is probably something in the Docker best practices that make an outside of repository about the sourcing of an unversioned script to build an image. I need to look it up. For the moment a REPOSITORY/KEY static argument to setup-custom-repository.sh  to do the install.

Reply:
probably we do not need to make `SETUP_SCRIPT` configurable (the different script is needed to Enterprise, but Enterprise uses different Docker file)